Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -380,7 +385,7 @@ public void AddManager(Type type, IMixedRealityManager manager)
             else
             {
                 MixedRealityComponents.Add(new Tuple<Type, IMixedRealityManager>(type, manager));
-                manager.Initialize();
+                if(!isMixedRealityManagerInitializing) manager.Initialize();
```

Comment:
braces

Reply:
also, seeing as this is a singleton, how are we initializing more than once at a time?

Reply:
It's not the singleton that was being initialized twice, is was the Runtime DeviceManager.

In MRManager.Initialise, we add managers and then call InitializeAllManagers.
BUT, when you add a RunTime manager, we also initialize it (as it wasn't expected to add runtime managers during MRManager initialization, now we do)

SO we need to ensure Runtime managers added on MixedRealityManager startup don't initialize themselves.  But it's needed at runtime of course.